extensions/amp-sidebar/1.0/toolbar.js

```diff
@@ -100,36 +99,22 @@ export class Toolbar {
    * @private
    */
   buildCallback_() {
-    this.toolbarClone_ = this.toolbarDOMElement_.cloneNode(true);
-    const targetId = this.toolbarDOMElement_.getAttribute('target');
+    this.toolbarClone_ = this.toolbarDomElement_.cloneNode(true);
+    const targetId = this.toolbarDomElement_.getAttribute('toolbar-target');
```

Comment:
So, here, `user().assert(...getAttribute('toobar-target'), '"toolbar-target" is required')`